modules/RTC/JitsiLocalTrack.js

```diff
@@ -442,11 +444,11 @@ export default class JitsiLocalTrack extends JitsiTrack {
                     this._startStreamEffect(this._streamEffect);
                 }
 
-                this.containers.map(
-                    cont => RTCUtils.attachMediaStream(cont, this.stream));
+                this.containers.map(cont => RTCUtils.attachMediaStream(cont, this.stream));
 
                 return this._addStreamToConferenceAsUnmute();
-            });
+            })
+            .catch(error => Promise.reject(error));
```

Comment:
hmm why is this call necessary? isn't it equivalent to not catching?

Reply:
Oh, this was unintentional, I had added some debugging info here when I was testing error handling in the JM PR and I removed since it wasn't necessary to log it here, forgot to remove this line. I will remove this.